Makefile

```diff
@@ -76,12 +76,21 @@ endif
 DEPS_CMAKE_FLAGS ?=
 USE_BUNDLED ?=
 
+ifdef BUNDLED_CMAKE_FLAG
+  $(error BUNDLED_CMAKE_FLAG was removed. Use DEPS_CMAKE_FLAGS instead)
+endif
+
+ifdef BUNDLED_LUA_CMAKE_FLAG
+  $(error BUNDLED_LUA_CMAKE_FLAG was removed. Use DEPS_CMAKE_FLAGS instead)
+endif
+
+# If USE_BUNDLED is non-empty, prepend the flag to DEPS_CMAKE_FLAGS
 ifneq (,$(USE_BUNDLED))
```

Comment:
Also wondering why this `USE_BUNDLED` special case is needed. Why can't we just document how to pass `-DUSE_BUNDLED` as a flag?